frontend/src/styles/Home.css

```diff
@@ -164,6 +164,13 @@
   width: 100%;
 }
 
+.not-found-projects-text {
+  color: white;
+  font-size: 1.5rem;
+  text-align: center;
+  margin-top: 20px;
+}
+
```

Comment:
this too

Reply:
i already fixed this please refer to latest commits